spec/index.bs

```diff
@@ -1999,6 +1999,13 @@ interface GPUPipelineLayout {
 GPUPipelineLayout includes GPUObjectBase;
 </script>
 
+{{GPUPipelineLayout}} has the following internal slots:
+<dl dfn-type=attribute dfn-for="GPUPipelineLayout">
+    : <dfn>\[[bindGroupLayout]]</dfn> of type `sequence<GPUBindGroupLayout>`.
+    ::
+        The layout of {{GPUBindGroup}}s that {{GPUPipelineBase}} using `this` will be compatible with.
```

Comment:
can we say something that BGL at sequence index `i` is expected at binding slot `i`, referring to `set_bind_group` from here to avoid confusion?

Reply:
Definitely thanks for looking into it in #724!